### PR TITLE
Implement fixedmodel interface

### PR DESCRIPTION
**Pull request by odow**
Interface for the fixedmodel function in Gurobi.
http://www.gurobi.com/documentation/6.0/refman/grbfixedmodel.html

### CODE REVIEW

**Comment by mlubin** [comment]
Looks good to me. Is the user supposed to free this model with GRBfreemodel? The documentation doesn't make this clear.

File: src/grb_solve.jl [hunk 1]
```diff
@@ -18,6 +18,16 @@ function computeIIS(model::Model)
     nothing
 end
 
+function fixedModel(model::Model)
```
Comment (mlubin) [inline src/grb_solve.jl]: This should probably go in ``grb_model.jl`` instead of ``grb_solve.jl``.

**Comment by IainNZ** [comment]
Thanks for PR @odow. Is that software tool you made being used for allocating students in that 4th year sustainability class?

**Comment by odow** [comment]
moved to grb_model. @IainNZ it is. You should have seen their old spreadsheet for doing it...

File: src/grb_solve.jl [hunk 2]
```diff
@@ -18,7 +18,6 @@ function computeIIS(model::Model)
     nothing
 end
 
-
```
Comment (mlubin) [inline src/grb_solve.jl]: Could you tidy this up and squash the commits?

**Comment by odow** [comment]
@mlubin the user would have to free the fixed model once they are finished using it. The GRBcopy function behaves similarly in returning a new pointer.

**Comment by mlubin** [comment]
@odow, ``Model`` objects are automatically freed when the julia object is garbage collected: https://github.com/JuliaOpt/Gurobi.jl/blob/eb1320cd7988b4a0aba2785001719e41c69a1bcf/src/grb_model.jl#L19

One has to be careful to not use the ``finalize_env`` option in the case where an environment is used for more than one model.

**Pull request merged**